src/background.c

```diff
@@ -746,15 +746,15 @@ void draw_bg_models(Mtx viewFromWorld, struct StageBgModel *bgModels, int bgMode
             avdisp_draw_model_unculled_sort_all(model);
         }
         if (bgModels->unk34 != 0)
-            func_80055C6C(viewFromWorld, bgModels->unk34);
+            draw_bg_flipbooks(viewFromWorld, bgModels->unk34);
         u_avdisp_set_some_func_1(0);
         if (customLightGroup > 0)
             pop_light_group();
     }
 }
 
 // 890
-s16 lbl_801B9A08[] =
+s16 s_nightFlipbookModels65[] =
```

Comment:
You can include the background model ID headers from /data and use the enums here.